src/tools/rados/rados.cc

```diff
@@ -290,7 +290,7 @@ int trunc([[maybe_unused]] IoCtx& io_ctx, const std::string& oid, const uint64_t
   if (use_striper)
     return striper().trunc(oid, offset);
 #endif
- 
+
```

Comment:
in general, i'd suggest avoid massive reformatting, unless you happen to be changing this area.